app/src/sharedTest/java/org/oppia/app/profile/PinPasswordActivityTest.kt

```diff
@@ -117,7 +120,8 @@ class PinPasswordActivityTest {
         userId
       )
     ).use {
-      onView(withId(R.id.input_pin)).perform(typeText("123"))
+      onView(withId(R.id.input_pin)).perform(typeText("123"), closeSoftKeyboard())
+      SystemClock.sleep(1000)
```

Comment:
We never use this because this sleep can actually hide the actual issue that might be happening in PR.